README.md

```diff
@@ -64,6 +64,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@master
+    - name: Install SVN ( Subversion )
+      run: sudo apt-get install subversion  
```

Comment:
It is better to run `sudo apt-get update` before installing a package. As suggested in a note [here](https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/customizing-github-hosted-runners#installing-software-on-ubuntu-runners:~:text=Always%20run%20sudo%20apt%2Dget%20update%20before%20installing%20a%20package.%20In%20case%20the%20apt%20index%20is%20stale%2C%20this%20command%20fetches%20and%20re%2Dindexes%20any%20available%20packages%2C%20which%20helps%20prevent%20package%20installation%20failures.).

```suggestion
    - name: Install SVN ( Subversion )
      run: |
        sudo apt-get update
        sudo apt-get install subversion
```